examples/replication/outputs.tf

```diff
@@ -1,9 +1,9 @@
 output "source_bucket_name" {
   description = "Bucket Name"
-  value       = module.cos_source_bucket.bucket_name
+  value       = module.cos_source_bucket.bucket_names
 }
 
 output "target_bucket_name" {
   description = "Bucket Name"
-  value       = module.cos_target_bucket.bucket_name
+  value       = module.cos_target_bucket.bucket_names
```

Comment:
To get from bucket_names to a bucket_name, perhaps it should be bucket_names[0]. This would return a string rather than an array containing a string.